runtime/client/ext-helper.js

```diff
@@ -96,25 +97,12 @@ function launchApp (main, activity) {
 
 var aliveInterval
 function keepAlive () {
-  /**
-   * FIXME: though there do have listeners on process#message,
-   * ShadowNode still exits on end of current context.
-   * Yet this process should be kept alive and waiting for life
-   * cycle events.
-   */
+  if (aliveInterval) {
+    clearInterval(aliveInterval)
+  }
   aliveInterval = setInterval(() => {
     process.send({ type: 'ping' })
   }, 5 * 1000)
-  process.on('message', message => {
-    if (message.type === 'pong') {
-      logger.info('Received pong from VuiDaemon.')
-      stopAlive()
-    }
-  })
```

Comment:
How could we verify the alive process if removing the pong response?

Reply:
The pong response doesn't been used anyway. ANR were counted in runtime process and depended on the ping message sent from child process.

Reply:
Ok, if we remove the respond with PONG, I think the PING would be renamed to another like LIVE or ALIVE, that reports the application state. In general, the message PING represents to try to access a service.